shared/search/user-pane/index.js

```diff
@@ -49,7 +49,7 @@ export default connector.connect(
       return {
         mode: 'external',
         nonUserInfoProps: {
-          avatar: userForInfoPane.serviceAvatar,
+          avatar: userForInfoPane.extraInfo.serviceAvatar || '',
```

Comment:
i think in general this data structure in the store is a bit confusing as its very view centric and it really changes depending on the context. it might be easier to reason about if we had a flatter structure about the user like

```
export type SearchResult = {
  keybaseUsername: string,
  keybaseFullName: string,
  keybaseIsFollowing: boolean,
  serviceIcon: IconType,
  servieAvatar: string,
  serviceUsername: string,
  serviceFullName: SearchPlatforms,
  serviceProfileUrl: string,
}
```

then your logic for which data goes on which side would be handled by your renderers depending on the search platform. right now the data can switch sides and have different combinations, and the 'upgrade' code path also feels wonky. i'm fine to leave this for now but this seems simpler to me